presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/JtsGeometryUtils.java

```diff
@@ -181,6 +281,62 @@ private static Geometry readPolyline(SliceInput input)
         return GEOMETRY_FACTORY.createMultiLineString(lineStrings);
     }
 
+    /**
+     * serialize a geometry to a ESRI polyline.
+     * Note: both jts multilinestring and linestring corresponds to polyline.
+     *
+     * @param lineStrings A {@link LineString} or {@link MultiLineString}
+     */
+    private static void writePolyline(Geometry lineStrings,
+                                      SliceOutput output)
+    {
+        requireNonNull(lineStrings, "polyline is null");
+        requireNonNull(output, "output is null");
+
+        int partLengh = 0;
+        if (lineStrings.getGeometryType() == JTS_MULTI_LINESTRING) {
```

Comment:
"Use the string.equals(Object other) function to compare strings, not the == operator.

The function checks the actual contents of the string, the == operator checks whether the references to the objects are equal. Note that string constants are usually "interned" such that two constants with the same value can actually be compared with ==, but it's better not to rely on that."